src/Microsoft.DotNet.XHarness.CLI/CommandArguments/Android/Arguments/ShowAdbPathArgument.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.DotNet.XHarness.CLI.CommandArguments.Android
+{
+    internal class ShowAdbPathArgument : SwitchArgument
+    {
+        public ShowAdbPathArgument() : base("adb|show-adb-path", "Prints path to the adb executable XHarness is using", false)

```

Comment:
This isn't true; it also means in the current form that it skips the rest of the steps of the command that uses it?

Reply:
Updated to note that it does only that then